script/hassfest/translations.py

```diff
@@ -285,7 +285,7 @@ def gen_strings_schema(config: Config, integration: Integration) -> vol.Schema:
                     "user" if integration.integration_type == "helper" else None
                 ),
             ),
-            vol.Optional("config_entries_subentries"): cv.schema_with_slug_keys(
+            vol.Optional("config_subentries"): cv.schema_with_slug_keys(
```

Comment:
This reverts an accidental change introduced by https://github.com/home-assistant/core/pull/128157/commits/0dd8ac831a3a9afaa38496687b3dba9fe13f1a6f#diff-4df8655075a4b0fdc0f2d73b934be89066d4517a27bd529d81f5116583d23440